roles/rhel_subscribe/tasks/enterprise.yml

```diff
@@ -1,5 +1,10 @@
 ---
+- name: Disable all repositories
+  command: subscription-manager repos --disable="*"
+
 - name: Enable RHEL repositories
   command: subscription-manager repos \
                --enable="rhel-7-server-rpms" \
+               --enable="rhel-7-server-extras-rpms" \
+               --enable="rhel-7-server-optional-rpms" \
```

Comment:
Ok. I took the list of repos from the docs, and 'optional' was listed there. Just submitted openshift/openshift-docs#875 to update that, and pushing a new commit to update this pr... should I squash it?